docs/pages/docs/core-concepts/workspaces.mdx

```diff
@@ -0,0 +1,199 @@
+
+You should consider how to name your workspace in a monorepo, since this will be different than working in single repositories. To avoid npm naming collisions, you can use the name of the monorepo and name of the workspace, separated by a slash and preceded by an `@`. For example, `@todolist/api`.
+
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
+  <Tab>
+
+    Add the folders you want to configure as workspaces to the `workspaces` field in your root `package.json` file. This field contains a list of workspace folders in the form of globs:
+
+    ```json
+    {
+      "name": "monorepo",
+      "version": "1.0.0",
+      "workspaces": [
+        "docs",
+        "apps/*",
+        "packages/*"
+      ]
+    }
+    ```
+
+  </Tab>
+  <Tab>
+
+    Add the folders you want to configure as workspaces to the `workspaces` field in the root `package.json` file. This field contains a list of workspace folders in the form of globs:
+
+    ```json
+    {
+      "name": "monorepo",
+      "version": "1.0.0",
+      "workspaces": [
+        "docs",
+        "apps/*",
+        "packages/*"
+      ]
+    }
+    ```
+
+  </Tab>
+
+  <Tab>
+
+    Add the folders you want to configure as workspaces to the `pnpm-workspace.yaml` file that exists in your root directory. This file contains a list of workspace folders in the form of globs:
+
+    ```yaml
+    packages:
+      - "docs"
+      - "apps/*"
+      - "packages/*"
+    ```
+
+  </Tab>
+</Tabs>
+
+Each folder that matches any of the globs in the list is considered a workspace folder. In the examples, the `apps` and `packages` folders contain folders that are workspaces, and the `docs` folder itself is a workspace.
+
+```
+monorepo/
+├─ docs/
+├─ apps/
+│  ├─ api/
+│  ├─ mobile/
+├─ packages/
+│  ├─ tsconfig/
+│  ├─ config/
+├─ sdk/
+```
+
+Using the same workspace configuration, the `monorepo/docs/`, `monorepo/apps/api/`, `monorepo/apps/mobile/`, `monorepo/packages/config/`, and `monorepo/packages/tsconfig/` folders are all considered workspaces.
+The `monorepo/sdk/` folder is not considered a workspace and it is not included in the workspace configuration.
+
+## Managing workspaces
+
+When you move, delete, or rename your workspaces, you will have to make sure that all folders linked within your `package.json` matches. Anytime you change the configuration of your workspace, make sure you all dependencies of the workspace are also updated. Re-run your npm client's install command to check your configuration. If there are any problems after that, you may have to delete your `node_modules` folder and run an install again.
+
+## Managing dependencies
+
+To manage dependencies within workspaces in your monorepo, you will need to run commands that will only manage the dependencies of each workspace rather than for entire monorepo.
```

Comment:
```suggestion
To manage dependencies within workspaces in your monorepo, you will need to run commands that will only manage the dependencies of each workspace rather than for the entire monorepo.
```